plugins/dbviewer/frontend/public/javascripts/countly.views.js

```diff
@@ -32,12 +33,78 @@ window.DBViewerView = countlyView.extend({
 		else {
 			this.renderMain();
 		}
+		
+		var prepareSwitch = function() {
+			// check is not exist dbviewer_selected_app in localStorage
+			if (!store.get('dbviewer_selected_app')) {
+				var app_name = $.i18n.map["common.all"]; 
+				store.set('dbviewer_selected_app_name',app_name);
+			} 
+			// clear app-list
+			$('#app-list').html("");
+			// prepend "all apps" link to list
+			$('#app-list').prepend('<div data-value="all" class="app-option item" data-localize=""><span class="app-title-in-dropdown">'+$.i18n.map["common.all"]+'</span></div>');
+			// append list items
+			for (var key in countlyGlobal.apps) {
+				if (store.get('dbviewer_selected_app') && (countlyGlobal.apps[key]._id+"" === store.get('dbviewer_selected_app'))) {
+					app_name = countlyGlobal.apps[key].name; 
+					store.set('dbviewer_selected_app_name',countlyGlobal.apps[key].name);
```

Comment:
Let's not store app name, it may change, be edited, etc.
We store app_id and that is enough, we can always get current up to date app name from
`countlyGlobal.apps[app_id].name`